.github/workflows/scripts/cli-test.ps1

```diff
@@ -1,13 +1,26 @@
-$ErrorActionPreference = "continue"
-$CurrentPath = Get-Location
+$ErrorActionPreference = "stop"
+# Disable Telemetry:
+$env:NAB_DISABLE_TELEMETRY = "true"
```

Comment:
Is this for running locally? (should already have been set by the workflow)

Reply:
yes, when running locally.